app/controllers/collections_controller.rb

```diff
@@ -790,4 +803,16 @@ def reindex_items_if_necessary(collection_results)
     EOL::Solr::CollectionItemsCoreRebuilder.reindex_collection_items_by_ids(collection_item_ids_to_reindex.uniq)
   end
 
+  def limit_collection
+     if current_user.newish? || (current_user.newish? && current_user.assistant_curator?)
+       collections = current_user.collections.where(created_at: DateTime.now.to_date.beginning_of_day..DateTime.now.to_date.end_of_day)
+       if collections.any? {|x| x[:special_collection_id] != 2}
```

Comment:
Can we make this in another function?
def is_watch_list?
   ...
end
then add the check here